### PR TITLE
Cybereason malops fix

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18605

## Description
Filtering options weren't passed to querying function. A small fix. 

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
cybereason-packaging | https://github.com/demisto/content/pull/4137

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Integrations/Cybereason/Cybereason.yml [hunk 1]
```diff
@@ -552,8 +728,10 @@ script:
     - contextPath: Cybereason.User.Organization
       description: User organization
       type: string
-    description: Query users as part of investigation
+  dockerimage: demisto/python
```
Comment (Itay4) [inline Integrations/Cybereason/Cybereason.yml]: why did you add this ??

Comment (orlichter1) [reply Integrations/Cybereason/Cybereason.yml]: I'm getting an issue in Circle which I'm not sure why it's happening. Thought it was trying to use docker with python 3. Turns out it wasn't the cause

File: Releases/LatestRelease/Cybereason.md [hunk 2]
```diff
@@ -0,0 +1 @@
+- Query malops command using withinLastDays parameter issue fixed.
```
Comment (yaakovi) [inline Releases/LatestRelease/Cybereason.md]: remove this file

**Comment by orlichter1** [comment]
@kirbles19 Could you please go over the [release notes](https://github.com/demisto/content/edit/Cybereason-malops-fix/Integrations/Cybereason/CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F4138)?

**Comment by richardbluestone** [comment]
Hi, I've reviewed the mk file. Thanks

File: Integrations/Cybereason/Cybereason.py [hunk 3]
```diff
@@ -0,0 +1,1477 @@
...
+    machine_guid = get_machine_guid(machine_name)
+    procceses = query_processes(machine_name, file_name)
+    process_data = procceses['data'].get('resultIdToElementDataMap')
+    if not process_data:
+        return_error('Could not find process')
+    processes = process_data.keys()
+    for process_guid in processes:
+        response = kill_process(malop_guid, machine_guid, process_guid)
+        status_log = response['statusLog'][0]
+        status_log['status']
```
Comment (yaakovi) [inline Integrations/Cybereason/Cybereason.py]:
```suggestion
        status = status_log['status']
```

File: Integrations/Cybereason/Cybereason.py [hunk 4]
```diff
@@ -0,0 +1,1477 @@
...
+        # response
+        demisto.results('Request to kill process {0} was sent successfully and now in status {1}'.format(process_guid,
+                                                                                                         status_log))
```
Comment (yaakovi) [inline Integrations/Cybereason/Cybereason.py]:
```suggestion
                                                                                                         status))
```

File: Integrations/Cybereason/Cybereason.py [hunk 5]
```diff
@@ -0,0 +1,1477 @@
...
+        res = session.request(
+            method,
+            SERVER + url_suffix,
+            headers=headers,
+            data=data,
+            json=json,
+            verify=USE_SSL
+        )
+        if res.status_code not in {200, 204}:
+            return_error('Your request failed with the following error: ' + res.content + str(res.status_code))
```
Comment (yaakovi) [inline Integrations/Cybereason/Cybereason.py]: use `return_error` only in the main function

File: Integrations/Cybereason/Cybereason.py [hunk 6]
```diff
@@ -0,0 +1,1477 @@
...
+    except Exception, e:
+        LOG(e)
+        return_error('Your request failed with the following error: ' + res.content + str(res.status_code))
```
Comment (yaakovi) [inline Integrations/Cybereason/Cybereason.py]: same here and for the rest

File: Integrations/Cybereason/Cybereason.py [hunk 7]
```diff
@@ -0,0 +1,1477 @@
...
+            method,
+            SERVER + url_suffix,
+            headers=headers,
+            data=data,
+            json=json,
+            verify=USE_SSL
+        )
+        if res.status_code not in {200, 204}:
+            return_error('Your request failed with the following error: ' + res.content + str(res.status_code))
+    except Exception, e:
```
Comment (yaakovi) [inline Integrations/Cybereason/Cybereason.py]: no need to catch exceptions here since they will be handled in the main (code doesn't do anything special with the error).

File: Integrations/Cybereason/Cybereason.py [hunk 8]
```diff
@@ -0,0 +1,1477 @@
...
+    return response
+
+
+def add_comment_command():
+    comment = demisto.args().get('comment')
+    malop_guid = demisto.args().get('malopGuid')
+    try:
+        add_comment(malop_guid, comment.encode('utf-8'))
+        demisto.results('Comment added successfully')
+    except Exception, e:
```
Comment (yaakovi) [inline Integrations/Cybereason/Cybereason.py]:
```suggestion
    except Exception as e:
```

File: Integrations/Cybereason/Cybereason.py (hunk 5 above)
Comment (orlichter1) [reply Integrations/Cybereason/Cybereason.py]: What do you mean? how should I return in then?

**Comment by anara123** [comment]
@Itay4 @yaakovi what is status here?

**Comment by anara123** [comment]
@orlichter1 you have conflict here
